lib/py/src/transport/TSSLSocket.py

```diff
@@ -47,8 +47,12 @@ class TSSLBase(object):
     # SSL 2.0 and 3.0 are disabled via ssl.OP_NO_SSLv2 and ssl.OP_NO_SSLv3.
     # For python < 2.7.9, use TLS 1.0 since TLSv1_X nor OP_NO_SSLvX is
     # unavailable.
-    _default_protocol = ssl.PROTOCOL_TLS_CLIENT if _has_ssl_context else \
-        ssl.PROTOCOL_TLSv1
+    if sys.version_info < (3, 6):
+        _default_protocol = ssl.PROTOCOL_SSLv23 if _has_ssl_context else \
+            ssl.PROTOCOL_TLSv1
```

Comment:
I do wish appveyor can use an actual supported version of python 😞

can you link some doc explaining that on python <3.6 we can only use ssl v2/v3 over tls? (similar to the ipv6 one below but in comment form)

Reply:
> can you link some doc explaining that on python <3.6 we can only use ssl v2/v3 over tls? (similar to the ipv6 one below but in comment form)

I can try to add something that makes sense